src/App.tsx

```diff
@@ -1,7 +1,26 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export const App: React.FC = () => {
+  const [selectedKey, setSelectedKey] = useState('');
+
+  const handleKey = (event: KeyboardEvent) => {
+    setSelectedKey(event.key);
+  };
+
+  useEffect(() => {
+    document.addEventListener('keyup', handleKey);
+  }, []);
```

Comment:
don't forget to cleanup before component unmounts - a callback after `return` statement

```suggestion
  useEffect(() => (
    document.addEventListener('keyup', handleKey)
    
    return () => document.removeEventListener('keyup', handleKey);
  ), []);
```